rllib/connectors/learner/compute_returns_to_go.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, List, Dict
+
+import scipy
+
+from ray.rllib.connectors.connector_v2 import ConnectorV2
+from ray.rllib.core.columns import Columns
+from ray.rllib.core.rl_module.multi_rl_module import MultiRLModule
+from ray.rllib.utils.typing import EpisodeType
+
+
+class ComputeReturnsToGo(ConnectorV2):
+    """Learner ConnectorV2 piece computing discounted returns to go till end of episode.
+
+    This ConnectorV2:
+    - Operates on a list of Episode objects (single- or multi-agent).
+    - Should be used only in the Learner pipeline as a preparation for an upcoming loss
+    computation that requires the discounted returns to go (until the end of the
+    episode).
+    - For each agent, for each episode and at each timestep, sums up the rewards
+    (discounted) until the end of the episode and assigns the results to a new
+    column: RETURNS_TO_GO in the batch.
```

Comment:
`RETURNS_TO_GO` irritated me a bit as I was used to "rewards-to-go" in PG.

Reply:
"Returns to go" is more accurate. A reward is a single value for one timestep. Return is always the (maybe discounted) sum of remaining rewards till the end of the episode.
But yeah, with the plural ("rewards" vs "reward"), it might work. Let's consequently use "return" for "sum of rewards", like we already do in "episode_return_mean" (we used to call it "episode_reward_mean", which was highly misleading, b/c it could have meant the average single reward received per timestep).